hack/olm-registry/olm-artifacts-template.yaml

```
@@ -45,3 +47,5 @@ objects:
       env:
       - name: FEDRAMP
         value: "${FEDRAMP}"
+      - name: HOSTED_ZONE_ID
+        value: ${ZONE_ID}
```

Should be HOSTED_ZONE_ID
```suggestion
        value: ${HOSTED_ZONE_ID}
```